addons/mail/static/src/components/composer/composer_tests.js

```diff
@@ -85,8 +85,15 @@ QUnit.test('composer text input: basic rendering', async function (assert) {
     );
     assert.strictEqual(
         document.querySelector(`.o_ComposerTextInput_textarea`).placeholder,
-        "Write something...",
-        "should have placeholder in note editable of composer text input"
+        "Send a message to followers...",
+        "should have 'Send a message to followers...' as placeholder in note editable of composer text input"
```

Comment:
"note editable" what does that mean?

Reply:
I think it's a legacy, there was an element with note-editable as a class if I remember well.
I'll improve that comment